lib/ansible/modules/storage/glusterfs/gluster_heal_facts.py

```diff
@@ -0,0 +1,191 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright: (c) 2016, Red Hat, Inc.
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: gluster_heal_facts
+short_description: Gather facts about self-heal or rebalance status
+author: "Devyani Kota (@devyanikota)"
+version_added: "2.8"
+description:
+  - Gather facts about either self-heal or rebalance status.
+options:
+  name:
+    description:
+      - The volume name.
+    required: true
+    aliases: ['volume']
+  status_filter:
+    default: "self-heal"
+    choices: ["self-heal", "rebalance"]
+    description:
+      - Determines which facts are to be returned.
+      - If the C(status_filter) is C(self-heal), status of self-heal, along with the number of files still in process are returned.
+      - If the C(status_filter) is C(rebalance), rebalance status is returned.
+        self-heal, status of self-heal, along with the number of files still
```

Comment:
This description seems redundant to me.

Reply:
@Akasurde removed the redundant description. Please review.